lib/gui/pages/finish/templates/success.tpl.html

```diff
@@ -15,18 +15,18 @@ <h3 class="title"><span class="tick tick--success space-right-tiny"></span> Flas
     <div class="box center">
       <div class="fallback-banner">
         <div class="caption caption-big">Thanks for using
-          <svg-icon path="'../../assets/etcher.svg'"
+          <svg-icon path="'../assets/etcher.svg'"
```

Comment:
Hmmm, neither this html file nor the svg files it points to have moved, so why has the relative path changed here? Was it incorrect before?